birdhouse/default.env

```diff
@@ -25,7 +27,8 @@ export BASH_IMAGE="bash:5.1.4"
 export DATA_PERSIST_ROOT="/data"
 
 # Folder on the host to persist Jupyter user data (noteboooks, HOME settings)
-export JUPYTERHUB_USER_DATA_DIR="${DATA_PERSIST_ROOT}/jupyterhub_user_data"
+# Must use single-quote for delayed eval.
+export JUPYTERHUB_USER_DATA_DIR='$DATA_PERSIST_ROOT/jupyterhub_user_data'
```

Comment:
Can we leave the `{}` for better readability (here and everywhere else where quotes were changed)? It is easier to spot variables with them (`$` by itself is too easily confused with `S` depending on font style/size).

Reply:
done